vector/src/main/java/im/vector/app/features/widgets/WidgetFragment.kt

```diff
@@ -72,6 +72,7 @@ class WidgetFragment @Inject constructor() :
 
     private val fragmentArgs: WidgetArgs by args()
     private val viewModel: WidgetViewModel by activityViewModel()
+    private val permissionUtils = WebviewPermissionUtils()
```

Comment:
if we want to inject this via dagger (not really needed whilst the `WebviewPermissionUtils` has no dependencies of its own) we can include the instance via the injectable constructor of the fragment 

```kotlin
WidgetFragment @Inject constructor() 
```

becomes...

```kotlin
WidgetFragment @Inject constructor(
  private val permissionUtils: WebviewPermissionUtils
) 
```

and the `WebviewPermissionUtils` itself will need to be marked as injectable

```kotlin
class WebviewPermissionUtils @Inject constructor()
```

Fragments in the Element are injectable because of the [fragment module](https://github.com/vector-im/element-android/blob/develop/vector/src/main/java/im/vector/app/core/di/FragmentModule.kt#L857)

Reply:
Ah nice! Thanks a lot for explaining. Made the change to be a good citizen and help remember for next time. 🙂

Reply:
future us will appreciate the injectable boilerplate being done :smile: